src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -74,6 +76,15 @@
 	public String biomeCategory;
 	public BiomeEntry parent;
 
+	public boolean spawnStronghold;

```

Comment:
Add test case for these in https://github.com/MCreator/MCreator/blob/5fa77237a5f1512baf9d1f42dab8cd92a815a998/src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java#L204

Reply:
For boolean fields, use _true or negation of this to fill with test data, check how it is done in existing fields.